coapclient.py

```diff
@@ -95,7 +95,7 @@ def main():  # pragma: no cover
     content_type = None
     #ct = {'content_type': defines.Content_types["application/link-format"]}
     ct = {}
-    ct['accept'] = 0
+    ct['accept'] = 10000
```

Comment:
yeah, we need this, but we should also configure this somehow.
e.g. this should not be hard coded

Reply:
This is another unused file that I messed around with... I configured this inside my autoctt_client.py (line 28), I guess that should be fine?